azure-pipelines-integration.yml

```diff
@@ -18,7 +18,10 @@ jobs:
 - job: VS_Integration
   pool:
     name: NetCorePublic-Pool
-    queue: $(queueName)
+    queue: BuildPool.Windows.VS2019.Pre.Scouting.Open
+# One this integration test fix has flowed in to master-vs-deps, we can update the queueName variable, and revert this change.
+# Tracked by https://github.com/dotnet/roslyn/issues/51312
+#    queue: $(queueName)
```

Comment:
So we have a variable that controls everything for all branches, but we can't change it now because that'll break other branches, so we're doing this?

Reply:
yes